.circleci/config.yml

```diff
@@ -59,10 +59,19 @@ jobs:
           docker_layer_caching: true
       - run:
           command: |
+            mkdir -p coverage-standalone-27 && \
             docker container create --name dummy -v shared_vol:/app alpine && \
```

Comment:
@czerwingithub, I tried to run a job without this `dummy` container lines, and nothing changed.  Maybe we can remove those lines?

Reply:
Eh, let's leave them for now.... I don't fully understand these tests right now.  Just add a comment saying you believe they can be removed.